src/pdf/components.js

```diff
@@ -18,6 +18,6 @@ export const NumberedContent = ({
 
 export const CheckHere = ({text}: {text: string}) => (
   <div className="check-here">
-    (<span className="underline">{text}</span>)
+    <div className="underline">({text})</div>
```

Comment:
nit: I try to avoid nesting tags directly under `div`. I think `span` is a marginally better choice but a text tag with some semantic meaning is even better

Reply:
Fair. span was bringing in some baggage for me, I think. Using span, I couldn't fix the undesired newline break. This is what I refer to that there may be a more proper formatting method via CSS. Maybe with a check-here-underline class that has the right options. I messed with it for a bit but this was my best option thus far. Can be reworked.